### ISSUE TITLE
trailing text column flag t take modifiers that allow selection of words

### ISSUE THREAD

**Issue by MarceloBanik** [GenericMappingTools/gmt#1063]
**Description of the desired feature**
It could be useful be able to choose one word from the trailing text.
E.g., -i1-3,t:2 would consider the trailing text string as a list of words and select the third word instead of the whole thing. (Thanks Paul for the suggestion)

Other possibility is using -i as usual to select columns and indicate that there is a text with t, and then use -f to indicate that one of the columns is a text , using perhaps s to indicate a string, (as t is already used as relative time). Something like:
```
echo "2019-06-30T12 TEXT -23:30 -100:30" | gmt convert -i0,2,3,1,t -fi0T,1y,2x,3s -o2,1,0,3
-100.5  -23.5   2019-06-30T12:00:00  TEXT
```
Perhaps it is stretching too much, but note that the following command that skip the text column already works (eventually shouldn't):
```
echo "2019-06-30T12 TEXT -23:30 -100:30" | gmt convert -i0,2,3,1,t -fi0T,1y,2x -o2,1,0                                                                                                    
-100.5  -23.5   2019-06-30T12:00:00
```

**Are you willing to help implement and maintain this feature?** Yes/No
Unfortunately no, but I will spend some time trying to see how much I could contribute. The last time that I taken a look in the code it appeared beyond my comprehension.

**Comment by PaulWessel** [comment]
So I already implemented **t**[_word_] in **-i** and **-o**; it is pending approval (#1051), so there will be no colons, just a single integer starting at 0.  The fact that your second case "words" is that you are not plotting that column which would be NaN since not a numerical value.  Your input is violating the GMT data record format of all numerical columns first, followed by optional trailing text.

**Comment by seisman** [comment]
Closed as #1051 was merged.
